extensions/ql-vscode/src/remote-queries/view/StarCount.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { StarIcon } from '@primer/octicons-react';
+
+type Props = { starCount?: number };
+
+const StarCount = ({ starCount }: Props) => (
+  Number.isFinite(starCount) ? (
+    <>
+      <span className="vscode-codeql__analysis-star">
```

Comment:
How come you're using CSS classes here? My aim so far with these components is that they would be re-usable and self-contained. We can use StyledComponents and inline styles.

 E.g. 

```ts
const Star = styled.span`
  ...
`;

const Count = styled.span`
 ...
`;
```